modules/indexer/code/search.go

```diff
@@ -86,7 +87,22 @@ func HighlightSearchResultCode(filename, language string, lineNums []int, code s
 	return lines
 }
 
-func searchResult(result *internal.SearchResult, startIndex, endIndex int) (*Result, error) {
+func RawSearchResultCode(filename, language string, lineNums []int, code string) []*ResultLine {
```

Comment:
Should this function be private as it is only used in this file?

Reply:
Agreed as it shouldn't be used outside of this file.